lib/puppet_litmus/inventory_manipulation.rb

```diff
@@ -45,13 +48,11 @@ def localhost_inventory_hash
   # @param targets [Array]
   # @return [Array] array of targets.
   def find_targets(inventory_hash, targets)
-    require 'bolt/inventory'
-    if targets.nil?
-      inventory = Bolt::Inventory.new(inventory_hash, nil)
-      targets = inventory.node_names.to_a
-    else
-      targets = [targets]
-    end
+    targets = if targets.nil?
+                inventory_hash.to_s.scan(%r{uri"=>"(\S*)"}).flatten
```

Comment:
Need to flag this up with the bolt team to find new API to replace `Bolt::Inventory.new(inventory_hash, nil)` with a v2 inventory.